docs/complex-types/methods.md

```diff
@@ -0,0 +1,41 @@
+# methods
+
+In this section we will explore 🚀 all nooks and crannies behind methods types.
+
+
+
+Diving into `methods.ts` we have these three imports
+
+```ts
+import { EntitySchema } from "./data" // Record<string, any>
+import { SyncKey } from "./sync" // Opaque<string, "sync-key">
+import { Except } from "type-fest" // Create a type from an object type without certain keys.
+```
```

Comment:
Zastanawiam się czy tego nie pominąć, jeśli już to informacje o tych typach powinny być inline w tekście w którym natrafiasz na danych typ - albo w [ramce info](https://vitepress.dev/guide/markdown#custom-containers)